ui/helpers/utils/permission.js

```diff
@@ -405,6 +405,13 @@ export const PERMISSION_DESCRIPTIONS = deepFreeze({
     rightIcon: null,
     weight: 3,
   }),
+  [EndowmentPermissions['endowment:keyring']]: ({ t }) => ({
+    label: t('permission_keyring'),
+    description: t('permission_keyringDescription'),
+    leftIcon: getLeftIcon(IconName.UserCircleAdd),
+    rightIcon: null,
+    weight: 3,
```

Comment:
why is this weight 3? Seems like this and `snap_mangeAccounts` should be a 1.

Reply:
It's 3 here just because `snap_manageAccounts` was also 3 and they should stay close to each other. But I don't know why `snap_manageAccounts` was 1 in the first place.

Reply:
Can we change both of them to 1?

Reply:
Sure, on it 👍

Reply:
Ok, I checked the Notion page and it was supposed to be 2, actually. I imagined it's not a one because:

> Medium. The Snap is managing accounts on behalf of the user. But these are new accounts. The Snap is not accessing existing accounts. The risk has to do with whether the Snap itself is well designed enough to mitigate potential risk factors with key management.

So I switched both to two now.

Reply:
Ok spoke with Christian to get some context behind the decision. I understand the permission being a 2 because it's a fresh key, but I think a discussion around surfacing the gravity of this permission to the user (even though it is not a 1) is warranted in the future.